accessibility_monitoring_platform/apps/cases/migrations/0093_populate_case_numbers.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 5.0.4 on 2024-04-16 13:16
+
+from django.db import migrations
+
+
+def populate_case_numbers(apps, schema_editor):  # pylint: disable=unused-argument
+    Case = apps.get_model("cases", "Case")
+    for case in Case.objects.all():
+        case.case_number = case.id
+        case.save()
```

Comment:
I think we need to give the case number their correct sequential number. Can we rely on the iterator to show them in the correct order?

Reply:
I do not understand your comment.

What do you mean by 'correct' if not the `Case.id` which has been used up to now?

The order of cases returned in this loop makes no difference.

Chris said that he doesn't want the numbers of existing cases to change.